src/style.css

```diff
@@ -114,6 +118,361 @@ nav {
   margin-top: 32px;
 }
 
+#recent {
+  font-size: 250%;
+  padding-top: 18.9%;
+  padding-bottom: 5%;
+  padding-left: 9%;
+  font-family: "Crete Round", sans-serif;
+  color: #1c1a19;
+}
+
+.grid-container {
+  padding: 5px;
+  display: grid;
+  text-align: center;
+  row-gap: 4%;
+}
+
+.grid-item {
+  padding: 4px;
+}
+
+#rImg {
+  padding-top: 9%;
+  width: 85%;
+}
+
+.works {
+  background: none;
+  margin-left: -12px;
+}
+
+#dash {
+  width: 85%;
+  margin-left: 42px;
+  margin-top: 20px;
+  margin-bottom: 70px;
+}
+
+#paragraph-title {
+  font-size: 250%;
+  font-family: "Crete Round", sans-serif;
+  color: #091e42;
+  margin-left: -62px;
+}
+
+.skills {
+  display: flex;
+  justify-content: space-around;
+  padding-left: 5.5%;
+  padding-bottom: -15px;
+  width: 93%;
+  color: #091e42;
+}
+
+.skills li {
+  padding: 3.5%;
+  font-family: 'Inter', sans-serif;
+
+  /* border-style: solid; */
+  border-spacing: 4%;
+  border: 1px solid #8993a4;
+}
+
+#multi {
+  text-align: left;
+  font-size: 105%;
+  font-family: "Inter", sans-serif;
+  margin-left: 9%;
+  color: #091e42;
+
+  /* text-align: center; */
+}
+
+.see-button {
+  padding-right: 61%;
+}
+
+.prof {
+  padding-top: 25%;
+  color: #fff;
+  margin-left: 8%;
+
+  /* background: url(Img/Img\ Placeholder.png);
+  background-repeat: no-repeat;
+  background-position-x: center;
+  background-position-y: 150px; */
```

Comment:
Kindly, I suggest you remove all commented codes from your project to make it more professional and to follow the best practices. 😄